tests/chart_tests/test_probes.py

```diff
@@ -26,6 +26,22 @@ def test_template_probes_with_custom_values(self, doc):
             assert container["livenessProbe"] != {}
             assert container["readinessProbe"] != {}
 
+    filtered_docs = [doc for doc in docs if doc["kind"] in include_kind_list]
```

Comment:
We should probably give this a different name. I'm not sure on the ordering of how pytest sets these up, but overwriting the previous variable may lead to unexpected behavior. Let's switch the previous assignment to `filtered_containers` since that's what they are, not individual yaml docs produced by helm.